.coafile

```diff
@@ -46,3 +46,7 @@ language = python3
 # 0.8.x (current stable release) needs them.
 ci_keywords, keywords = \#TODO, \# TODO, \#FIXME, \# FIXME
 cs_keywords =
+
+[PythonPackageInit]
+bears = PythonPackageInitBear
```

Comment:
The reason I didn't do that was because I'm ignoring the python files in the root directory, I can't do that in the `python` section